Added punctuation to About Us Spanish Heading (#1130)
playfulprogramming/playfulprogramming: content/carlosrangel-dev/index.md

---
{
	name: "Carlos Rangel Dev",
	firstName: "Carlos",
	lastName: "Rangel",

> 	description: "Carlos is a FrontEnd developer that visits BackEnd world occasionally. 🤖\nHe is starting his adventure with code and looking to contribute to interesting projects and connect with people through coding and books. 🧙‍♂️\nHe likes to write for his blog sites, write short fiction, code, and spend time with his family.",

Reviewer: ```suggestion
	description: "Carlos is a FrontEnd developer who visits the BackEnd world occasionally. 🤖\nHe is starting his adventure with code and looking to contribute to interesting projects and connect with people through coding and books. 🧙‍♂️\nHe likes to write for his blog, write short fiction, code, and spend time with his family.",
```
